README.md

```diff
@@ -77,6 +81,10 @@ Run setup script
 
     bin/setup
 
+Install phantomjs 1.9.8 to run Javascript unit tests:
+
+    brew install phantomjs198
```

Comment:
@timurvafin, done in https://github.com/fs/rails-base/commit/9d44dd644eb52f4ca03ab0cdd39abe8cecb53316